### PR TITLE
Update to CurationConcerns 1.5.0

### LINKED ISSUE #175: Update to curation concerns 1.5.0

### PR DESCRIPTION
**Pull request by eliotjordan**
Updates to the latest version of CurationConcerns, Closes #175.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -6,16 +6,36 @@ end
...
-task spec: :rubocop do
-  RSpec::Core::RakeTask.new(:spec)
-end
+require 'active_fedora/rake_support'
+
 Dir.glob('tasks/*.rake').each { |r| import r }
 
 Bundler::GemHelper.install_tasks
 
+namespace :geo_concerns do
```
Comment (drh-stanford) [inline Rakefile]: can we move this into a `tasks/xxx.rake` file?

**Comment by drh-stanford** [comment]
Looks good to me, only one minor comment.

**Comment by jrgriffiniii** [comment]
Thank you for this.  The Coveralls reports seem optimistic, but I cannot see how there are any missing RSpec suites which would lead me to doubt such a high coverage.

File: Rakefile (hunk 1 above)
Comment (eliotjordan) [inline Rakefile]: Done.

**Comment by eliotjordan** [comment]
@drh @jrgriffiniii Moved some rake tasks around. How's it looking now?

**Pull request merged**